### PR TITLE
[incubator/gogs] Upgrade gogs image

### PR DESCRIPTION
**Pull request by obeyler**
#### What this PR does / why we need it:
just upgrade the gogs docker image

#### Special notes for your reviewer:
see https://github.com/gogs/gogs/releases/tag/v0.11.79

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

File: incubator/gogs/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 description: 'Gogs: Go Git Service'
 name: gogs
-version: 0.7.4
-appVersion: 0.11.66
+
```
Comment (gyliu513) [inline incubator/gogs/Chart.yaml:4]: I found most of the chart do not have such blank line for their Chart.yaml, do we need to keep this?

Comment (obeyler) [reply incubator/gogs/Chart.yaml:4]: @gyliu513 with do you want to remove version and appversion ?
If the chart.yaml doesn't have blank line circle ci is not happy ;-) and test failed

**Comment by obeyler** [comment]
@prydonius @responsible @tvon @sonnyvan @kevinschumacher @jpds @gyliu513 @fuhbar @cheyang @AdamDang
Could someone put lgtm label on this PR please ?
Thanks

**Comment by prydonius** [comment]
@obeyler what was the error you're getting from CircleCI? The empty line in the Chart.yaml should not be necessary.

**Comment by obeyler** [comment]
@prydonius  Ok chart.yaml accept to have no empty line at the end of file unlike value.yaml
I note that for the next time

File: incubator/gogs/Chart.yaml (hunk 1 above)
Comment (obeyler) [reply incubator/gogs/Chart.yaml:4]: @gyliu513 sorry I test it after Ok chart.yaml accept to have no empty line at the end of file unlike value.yaml
I note that for the next time

Comment (prydonius) [inline incubator/gogs/Chart.yaml:4]: this empty line is still here?

Comment (obeyler) [reply incubator/gogs/Chart.yaml:4]: I dont know why, I recreate a new PR with a clean fork

**Pull request closed without merging**